examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py

```diff
@@ -78,7 +78,40 @@
                                                                            this should align with your model config, \
                                                                            and your dataset builder args: args.pad_max_length')
 parser.add_argument('--gptq_debug', action='store_true', help='Whether to use debug model ')
-# =======================================
+# ==============code generation args===========
+parser.add_argument("--code_generation", action="store_false")
+parser.add_argument("--n_samples", default=200, type=int)
```

Comment:
Can we just add parameters that the script required?

Reply:
it is used with line 322, the parameters requested by bigcode-evaluation repo.
https://github.com/bigcode-project/bigcode-evaluation-harness